Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -37,6 +39,7 @@ import PayPalCheckout
         self.nativeCheckoutProvider = nativeCheckoutProvider
     }
 
+    @available(*, deprecated, message: "BraintreePayPalNativeCheckout Module is deprecated, use BraintreePayPal Module instead")
```

Comment:
We should move this one above the docstring instead of above the mark

Reply:
Great catch!